### PR TITLE
Feature/var msite

### PR DESCRIPTION
**Pull request by Sukumar-K7**
added nudge var  in partners and added field in navigationItem

### CODE REVIEW

**Comment by Sukumar-K7** [comment]
added nudge var for this component in partners
image

added aesthetic in nudge by vamsi

added small Thumbnail field in navigationItem for msite image.

File: schemas/ihcl-gift-cards/index.ts [hunk 1]
```diff
@@ -30,12 +30,18 @@ export const giftCards: FeatureSchemaDefinition = {
...
     ],
-  
+
     tabs: [
       {
         title: "[Gift Card] Manage Card Multiple Tabs",
         value: "giftCards.tabs.manage-card-multiple-tabs",
       },
     ],
+    stepper: [
```
Comment (NarsingRao13) [inline schemas/ihcl-gift-cards/index.ts:40]: please attach UI screen shots

File: schemas/ihcl-partners/index.ts [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
-        ]
-    }
-}
\ No newline at end of file
+  variants: {
+    group: [],
+    card: [
+      {
+        title: "[partners] Right Media Image Carousel with Left Content",
+        value: "partners.card.right-media-image-carousel-with-left-content",
```
Comment (NarsingRao13) [inline schemas/ihcl-partners/index.ts:9]: Please attach UI ss

File: package.json [hunk 3]
```diff
@@ -38,6 +38,8 @@
     "@sanity/scheduled-publishing": "^0.1.9",
     "@sanity/ui": "^0.37.1",
     "@sanity/vision": "^2.30.1",
+    "axios": "^1.3.4",
+    "js-base64": "^3.7.5",
```
Comment (NarsingRao13) [inline package.json:42]: Please remove unused dependencies from package.json file

Comment (Sukumar-K7) [reply package.json]: removed

File: schemas/ihcl-partners/index.ts (hunk 2 above)
Comment (Sukumar-K7) [reply schemas/ihcl-partners/index.ts:9]: image

File: schemas/ihcl-gift-cards/index.ts (hunk 1 above)
Comment (Sukumar-K7) [reply schemas/ihcl-gift-cards/index.ts:40]: image

Comment (Sukumar-K7) [reply schemas/ihcl-gift-cards/index.ts]: removed the var

**Pull request merged**